pages/common/virtualenv.md

```diff
@@ -1,6 +1,7 @@
 # virtualenv
 
 > Create virtual isolated Python environments.
+> Homepage: <https://virtualenv.pypa.io/en/latest/>.
```

Comment:
I think the `/en/latest/` is appended automatically when visiting https://virtualenv.pypa.io, so IMO we could simplify the link here.